thoth/metrics_exporter/jobs.py

```diff
@@ -76,63 +76,82 @@
+                    label_selector=label_selector, namespace=namespace
+                )
 
+                for j_status, j_counts in jobs_status_evaluated.items():
+                    metrics.jobs_status.labels(label_selector, j_status, namespace).set(j_counts)
 
-def get_configmaps_per_namespace_per_label():
-    """Get the total number of configmaps in the namespace based on labels."""
-    namespaces = get_namespaces()
+                _LOGGER.debug("thoth_jobs=%r", jobs_status_evaluated)
 
-    _OPENSHIFT = OpenShift()
-    for namespace in namespaces:
-        _LOGGER.info("Evaluating configmaps metrics for Thoth namespace: %r", namespace)
+    @staticmethod
+    def count_configmaps(config_map_list_items: list) -> int:
+        """Count the number of ConfigMaps for a certain label in a specific namespace."""
+        return len(config_map_list_items["items"])
 
-        for label in _JOBS_LABELS + ["operator=graph-sync", "operator=workload"]:
-            config_maps_items = _OPENSHIFT.get_configmaps(namespace=namespace, label_selector=label)
-            number_configmaps = count_configmaps(config_maps_items)
-            metrics.config_maps_number.labels(namespace, label).set(number_configmaps)
-            _LOGGER.debug(
-                "thoth_config_maps_number=%r, in namespace=%r for label=%r", number_configmaps, namespace, label
-            )
+    def get_configmaps_per_namespace_per_label(self):
+        """Get the total number of configmaps in the namespace based on labels."""
+        namespaces = self.get_namespaces()
 
+        _OPENSHIFT = OpenShift()
+        for namespace in namespaces:
 
-def get_ceph_results_per_type():
-    """Get the total number of results in Ceph per type."""
-    for store in _MONITORED_STORES:
-        if not store.is_connected():
-            store.connect()
-        all_document_ids = store.get_document_listing()
-        list_ids = [str(cid) for cid in all_document_ids]
-        metrics.ceph_results_number.labels(store.RESULT_TYPE).set(len(list_ids))
-        _LOGGER.debug(f"ceph_results_number for {store.RESULT_TYPE} ={len(list_ids)}")
+            for label in _JOBS_LABELS + ["operator=graph-sync", "operator=workload"]:
+                _LOGGER.info("Evaluating ConfigMaps(label_selector=%r) metrics for namespace: %r", label, namespace)
+                config_maps_items = _OPENSHIFT.get_configmaps(namespace=namespace, label_selector=label)
+                number_configmaps = self.count_configmaps(config_maps_items)
+                metrics.config_maps_number.labels(namespace, label).set(number_configmaps)
+                _LOGGER.debug(
+                    "thoth_config_maps_number=%r, in namespace=%r for label=%r", number_configmaps, namespace, label
+                )
+
+
+class CephMetrics:
+    """Class to evaluate Metrics for Ceph."""
+
+    def get_ceph_results_per_type(self):
+        """Get the total number of results in Ceph per type."""
+        for store in _MONITORED_STORES:
```

Comment:
These variables like `_MONITORED_STORES`, `_JOBS_LABELS`, .. can go to the corresponding classes.